packages/drivers/debugger/src/fluidDebuggerController.ts

```diff
@@ -101,7 +103,10 @@ export class DebugReplayController extends ReplayController implements IDebugger
 		}
 
 		const tree = await this.documentStorageService.getSnapshotTree(version);
-		const seq = await DebugReplayController.seqFromTree(this.documentStorageService, tree);
+		const seq = await DebugReplayController.seqFromTree(
+			this.documentStorageService,
+			instanceOfIPartialSnapshotWithContents(tree) ? tree.snapshotTree : tree,
```

Comment:
I feel like this change should not be here. If you do inheritance (not encapsulation), you will not need to change code like that. See my other comment above where types are defined

Reply:
Removed